### ISSUE TITLE
Dependencies should be documented in the README.md file

### ISSUE THREAD

**Issue by mithro** [voc/voctomix#4]
I think your dependencies at the moment are;

 * Python
 * gstreamer >1.5
 * python-gst
 * ffmpeg

**Comment by mithro** [comment]
Set up instructions I have so far are;
```
sudo apt-get build-dep gstreamer1.0-libav gstreamer1.0-plugins-base gstreamer1.0-plugins-bad gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly
wget http://cgit.freedesktop.org/gstreamer/gstreamer/plain/scripts/create-uninstalled-setup.sh
chmod a+x create-uninstalled-setup.sh
./create-uninstalled-setup.sh
```

More to come....

**Comment by MaZderMind** [comment]
Fixed in 2fb65a153127288c5c7be22339bd2c7a4018190e
